search/search_repository.go

```diff
@@ -265,9 +274,15 @@ func (r *GormSearchRepository) search(ctx context.Context, sqlSearchQueryParamet
 		}
 		db = db.Limit(*limit)
 	}
+	if len(types) > 0 {
+		db = db.Joins("JOIN work_item_types w1 on w1.name=w.type")
+		db = db.Joins("JOIN work_item_types w2 on w2.path like (w1.path || '%')")
+		db = db.Where("w.type in(?)", types)
```

Comment:
Wow, I didn't know you were allowed to pass in an array and have it automatically transform to `type1,type2,type3`. Nice